charts/seed-controlplane/charts/etcd/values.yaml

```diff
@@ -17,32 +17,37 @@ servicePorts:
 metrics: basic
 
 scaleUpStabilization:
-  duration: "5m"
-  minCpuChange:
-    value: "1"
-    percentage: 80
-  minMemChange:
-    value: 2G
-    percentage: 80
+  stabilizationDuration: "5m"
+  minChange:
+    cpu:
+      value: "1"
+      percentage: 80
+    memory:
+      value: 2G
+      percentage: 80
 
 scaleDownStabilization:
-  duration: "15m"
-  minCpuChange:
-    value: "1"
-    percentage: 80
-  minMemChange:
-    value: 2G
-    percentage: 80
+  stabilizationDuration: "15m"
+  minChange:
+    cpu:
+      value: "1"
+      percentage: 80
+    memory:
+      value: 2G
+      percentage: 80
 
-targetAverageUtilization: 80
+targetAverageUtilizationCpu: 80
+targetAverageUtilizationMemory: 80
+
+maintenanceWindow: {}
```

Comment:
Can you provide some example values here to show, what format these values would have?
Because AFAIK Shoots will always have a (at least random) maintenanceWindow, so we will probably always pass some values here, right?

Reply:
Thanks @tim-ebert for the review. Is that ok if we provide the sample data as a comment, so that even if we chose to remove maintenanceWindow requirement in the future, it doesn't impact here?

Reply:
Yes, thats ok for me